concordium-sdk/src/test/testresources/smartcontractschema/lib.rs

```diff
@@ -0,0 +1,242 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! # A Concordium V1 smart contract
```

Comment:
Please add some more documentation to this SC. 

For example, what is the purpose of it? Why is it here in the java sdk?